libs/coin-evm/docs/evm-family-integration-process/README.md

```diff
@@ -0,0 +1,106 @@
+3. Add an entry for the new currency in the `abandonSeedAddresses` (using the currency ID as key and `EVM_DEAD_ADDRESS` as value) under [`libs/ledgerjs/packages/cryptoassets/src/abandonseed.ts`](https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledgerjs/packages/cryptoassets/src/abandonseed.ts)
+4. Add the new currency ID to the `setSupportedCurrencies` function param on each relevant project ([CLI](https://github.com/LedgerHQ/ledger-live/blob/develop/apps/cli/src/live-common-setup-base.ts), [LLD](https://github.com/LedgerHQ/ledger-live/blob/develop/apps/ledger-live-desktop/src/live-common-set-supported-currencies.ts), [LLM](https://github.com/LedgerHQ/ledger-live/blob/develop/apps/ledger-live-mobile/src/live-common-setup.ts) and [LLC test environement](https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledger-live-common/src/__tests__/test-helpers/environment.ts))
+5. Add a new feature flag config for this currency:
+   1. The new feature flag type in the [`CurrencyFeatures`](https://github.com/LedgerHQ/ledger-live/blob/7513354754ce0326a4ebcbcd86a5b4b38898a49e/libs/ledgerjs/packages/types-live/src/feature.ts#L80-L124) type under [`libs/ledgerjs/packages/types-live/src/feature.ts`](https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledgerjs/packages/types-live/src/feature.ts)
+   2. The new feature flag definition with default value in the [`CURRENCY_DEFAULT_FEATURES`](https://github.com/LedgerHQ/ledger-live/blob/7513354754ce0326a4ebcbcd86a5b4b38898a49e/libs/ledger-live-common/src/featureFlags/defaultFeatures.ts#L23-L67) mapping under [`libs/ledger-live-common/src/featureFlags/defaultFeatures.ts`](https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledger-live-common/src/featureFlags/defaultFeatures.ts)
+   3. Use this new feature flag in both [LLD](https://github.com/LedgerHQ/ledger-live/blob/develop/apps/ledger-live-desktop/src/renderer/modals/AddAccounts/steps/StepChooseCurrency.tsx) and [LLM](https://github.com/LedgerHQ/ledger-live/blob/develop/apps/ledger-live-mobile/src/screens/AddAccounts/01-SelectCrypto.tsx) currencies entry points (add account / select crypto flow)
+6. Update related doc and snapshot files:
+   1. run `pnpm doc` under `libs/ledgerjs/packages/types-live` to update the doc
+   2. run `pnpm test:jest` under `apps/ledger-live-desktop` to update the snapshots
+7. Add the related currency icon as SVG format under the [`libs/ui/packages/crypto-icons/src/svg`](https://github.com/LedgerHQ/ledger-live/tree/develop/libs/ui/packages/crypto-icons/src/svg) folder within the [@ledgerhq/icons-ui](https://github.com/LedgerHQ/ledger-live/tree/develop/libs/ui/packages/crypto-icons) package
+   1. the file name should follow the `{currency_id}.svg` naming convention
+   2. the SVG icon should follow Ledger Live currency icon guideline (TBD)
```

Comment:
You can mention this tool, that's available to anyone and can be used to validate icon SVGs for LL.
https://live.ledger.tools/svg-icons

Reply:
Wasn't even aware of this tool's existence 😮 

Will update the doc to mention it 👍

Reply:
Done here 6572c4f6d6ac5cb5ba7f9a361e1b24198f03ddb0